source/fundamentals/crud/write-operations/bulk.txt

```diff
@@ -15,54 +15,73 @@ Bulk Operations
-perform multiple operations with one call to the database.
+The ``Collection`` and ``Client`` classes both provide a ``BulkWrite()``
+method. When calling ``BulkWrite()`` on a ``Collection`` instance, you can
+perform multiple write operations on a single collection. When calling
+``BulkWrite()`` on a ``Client`` instance, you can perform bulk writes across
+multiple namespaces. In MongoDB, a namespace consists of the database name and the collection
+name in the format ``<database>.<collection>``. 
+
+.. important:: Client Bulk Write Requirements
+
+   To perform bulk operations on a ``Client`` instance,
+   ensure that your application meets the following
+   requirements:
+
+   - Uses {+driver-short+} v2.1 or later 
+   - Connects to {+mdb-server+} v8.0 or later
 
 Sample Data
 ~~~~~~~~~~~
 
-The examples in this guide use the following ``Book`` struct as a model for documents
-in the ``books`` collection:
+The examples in this guide use the following structs:
+
+- ``Book`` struct, which models documents in the ``db.books`` collection.
+  Each document contains a description of a book that includes the title,
+  author, and page length.
+
+- ``Poem`` struct, which models documents in the ``db.poems`` collection.
+  Each document contains a description of a poem that includes the title,
+  author, and publication year.
 
 .. literalinclude:: /includes/fundamentals/code-snippets/CRUD/bulkOps.go
-   :start-after: start-book-struct
-   :end-before: end-book-struct
+   :start-after: start-structs
+   :end-before: end-structs
    :language: go
    :dedent:
 
 To run the examples in this guide, load the sample data into the
-``db.books`` collection with the following snippet:
+``books`` and ``poems`` collection by using the following snippet:
 
 .. literalinclude:: /includes/fundamentals/code-snippets/CRUD/bulkOps.go
    :language: go
    :dedent:
    :start-after: begin insertDocs
    :end-before: end insertDocs
 
-Each document contains a description of a book that
-includes the title, author, and page length corresponding to
-the ``title``, ``author``, and ``length`` fields in each document.
-
 .. include:: /includes/fundamentals/automatic-db-coll-creation.rst
 
-Bulk Write
-----------
+.. _golang-bulk-collection:
 
-To perform a bulk operation, pass an array of :ref:`WriteModel
-<golang-write-model>` documents to the ``BulkWrite()`` method.
+Collection Bulk Write
+---------------------
 
-Modify Behavior
-~~~~~~~~~~~~~~~
+To perform a bulk operation on a single namespace, call the ``BulkWrite()``
+method on a collection and pass an array of :ref:`WriteModel <golang-write-model-collection>`
+documents as a parameter.
 
-The ``BulkWrite()`` method optionally takes a ``BulkWriteOptions``
-type, which represents options you can use to modify its behavior. If
-you don't specify a ``BulkWriteOptions``, the driver uses the default
-values for each option.
+Modify Collection-Level Behavior
```

Comment:
S suggest moving the options and return value sections to the end of this section after the define write models section (applies to collection and client)